coral/src/app/features/approvals/acls/components/AclApprovalsTable.tsx

```diff
@@ -0,0 +1,159 @@
+import { useState } from "react";
+import {
+  GhostButton,
+  Icon,
+  StatusChip,
+  Table,
+  Flexbox,
+} from "@aivenio/aquarium";
+import { AclRequest } from "src/domain/acl/acl-types";
+import deleteIcon from "@aivenio/aquarium/dist/src/icons/delete";
+import infoSign from "@aivenio/aquarium/dist/src/icons/infoSign";
+import loadingIcon from "@aivenio/aquarium/dist/src/icons/loading";
+import tickCircle from "@aivenio/aquarium/dist/src/icons/tickCircle";
+import {
+  requestStatusChipStatusMap,
+  requestStatusNameMap,
+} from "src/app/features/approvals/utils/request-status-helper";
+
+type Props = {
+  aclRequests: AclRequest[];
+  activePage: number;
+  totalPages: number;
+  onDetails: (aclRequest: AclRequest) => void;
+  onApprove: (aclRequest: AclRequest) => void;
+  onDecline: (AclRequest: AclRequest) => void;
+};
+
+export default function AclApprovalsTable({
+  aclRequests,
+  activePage,
+  totalPages,
+  onDetails,
+  onApprove,
+  onDecline,
+}: Props) {
+  const [isLoading, setLoading] = useState<boolean>(false);
+  return (
+    <Table ariaLabel={`Acl requests, page ${activePage} of ${totalPages}`}>
+      <Table.Head>
+        <Table.Cell>Principals/Usernames</Table.Cell>
+        <Table.Cell>IP addresses</Table.Cell>
+        <Table.Cell>Topic</Table.Cell>
+        <Table.Cell>Environment</Table.Cell>
+        <Table.Cell>Team</Table.Cell>
+        <Table.Cell>ACL type</Table.Cell>
+        <Table.Cell>Status</Table.Cell>
+        <Table.Cell>Requested by</Table.Cell>
+        <Table.Cell>Requested on</Table.Cell>
+        <Table.Cell></Table.Cell>
+        <Table.Cell></Table.Cell>
+        <Table.Cell></Table.Cell>
+      </Table.Head>
+      <Table.Body>
+        {aclRequests.map((aclRequest) => (
+          <Table.Row key={aclRequest.req_no}>
+            <Table.Cell>
+              {aclRequest.acl_ssl?.map((ssl, index) => (
```

Comment:
This is missing the `Flexbox` wrapper to allow us to avoid the sticking between  `StatusChip` ^^
<img width="138" alt="Screenshot 2023-03-03 at 08 01 03" src="https://user-images.githubusercontent.com/20607294/222655685-09a51045-9056-40bd-8d00-9eabc95a8fb3.png">